packages/core/src/index.ts

```diff
@@ -33,10 +33,6 @@ export {
 } from "./ens/utils.js";
 
 export { SmartAccountProvider, noOpMiddleware } from "./provider/base.js";
-export type {
-  SmartAccountProviderConfig,
-  SmartAccountProviderOpts,
-} from "./provider/base.js";
```

Comment:
we should still be exporting the new types and/or schemas

Reply:
these are still exported as part of `export type * from "./provider/types.js";` since i moved these to the types folder